appdev/guides/contenthub.rst

```diff
@@ -0,0 +1,157 @@
+=============================================
+The content hub. Tips and tricks
+=============================================
+.. figure:: /_static/images/appdev/guides/contenthubimages/01.png
+        :align: center
+
+        Content Hub Share Page
+
+On Ubuntu Touch the apps are confined. The way of sharing files between them is through the Content Hub, a part of the system that takes care of file import, export and sharing.
+
+Different ways of sharing the content
+-------------------------------------
+As we can see in the `documentation <https://api-docs.ubports.com/sdk/apps/qml/Ubuntu.Content/ContentHandler.html#detailed-description>`_, there are several ways of handling the file to be shared:
```

Comment:
I prefer giving links a descriptive name, so here I would write `Content Hub documentation` as the link text.